opentracing-spring-web-autoconfigure/pom.xml

```diff
@@ -29,6 +29,13 @@
       <groupId>io.opentracing.contrib</groupId>
       <artifactId>opentracing-tracerresolver</artifactId>
       <version>${version.io.opentracing.contrib-opentracing-tracerresolver}</version>
+      <!-- TEMPORARY: until resolver updated, as it is pulling in 0.30.0 version of api -->
+      <exclusions>
+      	<exclusion>
+      	  <groupId>io.opentracing</groupId>
+      	  <artifactId>opentracing-api</artifactId>
+      	</exclusion>
+      </exclusions>
```

Comment:
Please replace this `exclusion` with an explicit `dependencyManagement` section in which the `opentracing-api` version is declared. That will also _upgrade_ the transitive dependency from tracerresolver instead of falsely excluding it.

Reply:
Yes could workaround the issue - but wouldn't it be better if the resolver doesn't transitively include the api? The resolver should just work with any api version associated with the tracer being used.

Reply:
Ignore - will use dependency management instead.